views/user/user-menu.html

```diff
@@ -0,0 +1,13 @@
+{% if ckan_user %}
+<ul id="user-menu">
+  <li class="user-profile-icon">
+    <a href="/profile">Hello, {{ ckan_user.name }}</a>
```

Comment:
Probably this is a partial that will be used in the `base.html` as part of navbar?